examples/custom-service-gen/build.gradle

```diff
@@ -1,44 +1,61 @@
 buildscript {
-    ext.kotlin_version = '1.3.11'
+    ext.kotlin_serialization_version = '0.20.0'
+    ext.pbandk_version = 'master-SNAPSHOT'
     repositories {
         mavenCentral()
     }
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.7'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.72'
+        classpath 'org.jetbrains.kotlin:kotlin-serialization:1.3.72'
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.12'
     }
 }
 
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 allprojects {
     apply plugin: 'kotlin'
 
     repositories {
         mavenCentral()
+        maven { url 'https://jitpack.io' }
+    }
+
+    dependencies {
+        implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
+    }
+
+    tasks.withType(KotlinCompile).configureEach {
+        kotlinOptions {
+            jvmTarget = '1.8'
+        }
     }
 }
 
 project(':application') {
+    apply plugin: 'kotlinx-serialization'
     apply plugin: 'application'
     apply plugin: 'com.google.protobuf'
 
     mainClassName = 'pbandk.examples.greeter.MainKt'
     applicationName = 'greeter'
 
     dependencies {
-        compile 'com.github.streem.pbandk:pbandk-runtime-jvm:0.8.0'
-        compile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0'
+        implementation "org.jetbrains.kotlinx:kotlinx-serialization-runtime:$kotlin_serialization_version"
+        implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.6'
+        implementation "com.github.streem.pbandk:pbandk-runtime-jvm:$pbandk_version"
```

Comment:
Worth centralizing the other version numbers like you did `pbandk_version`?

Reply:
Yep, will do